doc/stdlib/base.rst

```diff
@@ -156,6 +156,10 @@ Getting Around
 
    This is only needed if your module depends on a file that is not used via ``include``\ . It has no effect outside of compilation.
 
+.. function:: apropos(string)
+
+   .. Docstring generated from Julia source
+
```

Comment:
Did you push that branch?  I don't see it.  The error is simply:

```
$ ./julia doc/genstdlib.jl
Collect 1481 Docs

Converting stdlib/

KeyError(apropos)
WARNING: Mod Base apropos
INFO: no docs for apropos(string) in Base
```

I figured it was probably the submodule thing, but hadn't dug into it yet.  It'd be great to be able to put documentation in submodules (like Dates, FFTs, Docs, etc).